SKILL.tsv

```diff
@@ -3868,10 +3868,10 @@ SKILL_20150414_003867	Attack soul
 SKILL_20150414_003868	Basic attack on spirits when outside the body.
 SKILL_20150414_003869	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Strongly smack down enemy.
 SKILL_20150414_003870	Activate aura and increase HP recovery of allies nearby.
-SKILL_20150414_003871	Enemies attacked have chance of creating a magic circle of conversion.
+SKILL_20150414_003871	Creates a magic circle which has a chance of converting enemies.
 SKILL_20150414_003872	Temporarily increases the property resistances of you and your party members.
 SKILL_20150414_003873	Fire, Ice, Lightning Resistance: +#{CaptionRatio}# {nl}Poison, Earth Resistance: +#{CaptionRatio2}#{nl}Buff Duration: #{CaptionTime}# seconds
-SKILL_20150414_003874	May defeat mutant or devil-type enemy at once.
+SKILL_20150414_003874	May defeat mutant or devil-type enemies instantly.
```

Comment:
It's normally not a 100% chance of instakill.
So probably something like `Has a chance of instantly defeating Mutant or Devil-type enemies.`